mir_eval/separation.py

```diff
@@ -207,17 +209,12 @@ def bss_eval_sources(
         return (sdr[idx], sir[idx], sar[idx], popt)
     else:
         # return the default permutation for compatibility
-        popt = range(nsrc)
+        popt = list(range(nsrc))
```

Comment:
Whoops, sorry, I was too focused on fixing the iterator mistake... everything should be an ndarray.